certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/common/model/Recipient.java

```diff
@@ -2,4 +2,12 @@
 
 public record Recipient(RecipientId id, String name) {
 
+  public String getLogicalAddress() {
+    if (id.id().equals("FKASSA")) {
+      return "2021005521";
```

Comment:
Should this value be configurable(?) I believe the number as written on line 7 is used when we really want to send the messages to fk and not to a stub, as right now in sit2 for testing (and in prod). Supposedly this would correspond to the config property  SENDMESSAGETOFK_LOGICALADDRESS for old certs in webcert, and this value differs between environments. This link shows how the logical address was configured for sit2 to switch from sending messages to stub to instead send them to fk: https://inera.atlassian.net/wiki/spaces/IT/pages/315527922/SIT2+-+F+rs+kringskassan+-+Peka+om+FK+Stubbe